README.md

```diff
@@ -677,6 +677,10 @@ If _you_ have some interesting links / articles feel free to make a pull request
  - [V is for Vaporware](https://christine.website/blog/v-vaporware-2019-06-23)
  - [Asynchronous concurrency made simple](https://amphp.org/)
 
+### Week 27
```

Comment:
This link should be under this week (26)

Reply:
I thought that we'd end up to this week (27) before we'd see it merged so.. 😅